core/Dockerfile

```diff
@@ -13,10 +13,10 @@ RUN pip install -U pip
 RUN pip install --no-cache-dir -r requirements.txt
 
 COPY ./cat/plugins ./cat/plugins
-COPY ./install_plugin_dependencies.sh install_plugin_dependencies.sh
```

Comment:
corrected to 
`COPY ./install_plugin_dependencies.py install_plugin_dependencies.py`